docs/endpoints/posts.md

```diff
@@ -158,7 +158,7 @@ POST /api/v3/posts
 - **campaign_run_id**: (int)
   The drupal campaign run node id of the campaign that the user's post is associated with.
 - **type**: (string) required.
-  The type of post submitted. Must be one of the following types: photo, voter-reg, text, share-social.
+  The type of post submitted. Must be one of the following types: `photo`, `voter-reg`, `text`, `share-social`. `share-social` posts will be auto-accepted unless an admin sets a custom `status`.
```

Comment:
I'm not sure what this part means about an admin setting a custom status? Looks like it will be `accepted` no matter what if it comes in with type `share-social`

Reply:
@katiecrane great q! If an admin sends along a custom `status` when creating the post, the 
 custom `status` will be overridden [here](https://github.com/DoSomething/rogue/blob/master/app/Repositories/PostRepository.php#L101). Now looking at it more closely though, it seems like it will get set to `pending` if they don't set a custom status so we don't want that either! I'll update this now!

Reply:
ah I see now! yeah good catch, does look like it would be overwritten to `pending` ⚾️

Reply:
@katiecrane thanks for catching this! I updated the logic and also added a few more tests to make sure that if an admin is creating a `share-social` post with a custom `status`, then it will take that status instead of auto-accepting.